### PR TITLE
(maint) Update module using pdk 1.14.0

### PR DESCRIPTION
**Pull request by joshcooper**

### CODE REVIEW

**Comment by joshcooper** [comment]
```
$ bundle exec rake gettext:update_pot
Warning - msgcmp is not present on the system
String changes detected, replacing with updated POT file
$ gst
## update_pdk_1.14
 M locales/puppetlabs-augeas_core.pot
```

**Comment by kris-bosland** [comment]
Is `gst` == `git status`?
> ```
> $ gst
> ## update_pdk_1.14
>  M locales/puppetlabs-augeas_core.pot
> ```

File: .vscode/extensions.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "recommendations": [
+    "jpogran.puppet-vscode",
+    "rebornix.Ruby"
+  ]
+}
```
Comment (kris-bosland) [inline .vscode/extensions.json]: Interesting that `.vscode/` was added to .pdkignore but not .gitignore

**Comment by joshcooper** [comment]
yeah it's an alias I have to:

```
gst='git status -sb'
```

**Pull request merged**